src/index.html

```diff
@@ -1,13 +1,436 @@
+            <div class="ingredients__blok">
+              <ul class="ingredients__list">
+                <li class="ingredients__item">Kaolin</li>
+                <li class="ingredients__item">Sivak </li>
+                <li class="ingredients__item">Calcium</li>
+                <li class="ingredients__item">Licorice root</li>
+                <li class="ingredients__item">Turmeric</li>
+              </ul>
+            </div>
+            <a href="#" class="page__link ingredients__link">LEARN MORE</a>
+          </div>
+        </div>
+
+        <div class="ingredients__bottom">
+          <ul class="ingredients__advantages">
+            <li class="ingredients__advantages-item">
+              <img
+                src="./images/svg/snail.svg"
+                alt="слимак"
+                class="ingredients__advantages-snail"
+              >
+
+              <p class="ingredients__advantages-description">No synthetics and microplastics</p>
+            </li>
+
+            <li class="ingredients__advantages-item">
+              <img
+                src="./images/svg/recast.svg"
+                alt="переробка"
+                class="ingredients__advantages-recast"
+              >
+
+              <p class="ingredients__advantages-description">product packaging can be recycled or reused</p>
+            </li>
+
+            <li class="ingredients__advantages-item">
+              <img
+                src="./images/svg/leaf.svg"
+                alt="листя"
+                class="ingredients__advantages-leaf"
+              >
+
+              <p class="ingredients__advantages-description">only natural ingredients</p>
+            </li>
+          </ul>
+        </div>
+      </section>
+
+      <section class="contact container">
+        <div class="page__left ingredients__left">
+          <h3 class="page__left-title">Ingredients</h3>
+        </div>
+
+        <div id="contact" class="contact__element">
+          <h4 class="contact__title title">contact us</h4>
+
+          <p class="contact__help">We will help you choose a product that suits you and answer your other questions</p>
+
+          <form class="contact__form" action="#">
+            <input
+              class="contact__input"
+              type="text"
+              id="name"
+              name="name"
+              autocomplete="name"
+              placeholder="your name"
+              pattern=".{3,}"
+              required
+            >
+
+            <input
+              class="contact__input"
+              type="email"
+              id="email"
+              name="email"
+              autocomplete="email"
+              placeholder="e-mail"
+              pattern=".+@.+"
+              required
+            >
+
+            <input
+              class="contact__input"
+              type="tel"
+              id="phone"
+              name="phone"
+              autocomplete="tel"
+              placeholder="phone number"
+              pattern="{10}"
```

Comment:
The pattern for the phone number field in the form is incorrect. It should be pattern="\d{10}".